engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -855,6 +859,15 @@ struct shadow_word_pain_t final : public priest_spell_t
       }
 
       priest().refresh_insidious_ire_buff( s );
+
+      if ( child_searing_light && priest().buffs.divine_image->up() )
+      {
+        sim->print_debug( "searing_light triggered by shadow_word_pain" );
```

Comment:
same as above, i think instead of making a bunch of print statement telling you what procced searing light storing that inside of a proc or even a separate action per spell (like we do with psychic link) would feel better than adding a bunch of debug statements

Reply:
Resolved above

Reply:
@taherbert i still see this debug statement here btw, is that an error?